docs/dashboards/entity-dashboard.asciidoc

```diff
@@ -0,0 +1,94 @@
+[[detection-entity-dashboard]]
+= Entity dashboard
```

Comment:
Should this be "Entity Analytics dashboard" to match the dashboard's title in the UI?
```suggestion
= Entity Analytics dashboard
```